docs/releasenotes.md

```diff
@@ -7,7 +7,30 @@ title: Haal Centraal BRP Personen Bevragen
 
 ## Versie 2.0.0
 
-De property reisdocumenten is verwijderd uit "Persoon".
+- het request is gewijzigd van GET naar POST, waarbij alle parameters verplaatst zijn van de querystring in de url naar de requestbody
+- requestbody is polymorf gedefinieerd, waarbij in parameter 'type' bepaalt welke parameters gebruikt mogen en moeten worden
```

Comment:
Het raadpleeg en het zoeken endpoint is nu samengevoegd in 1 endpoint

Reply:
toegevoegd